src/Microsoft.Diagnostics.Monitoring.RestServer/Operation/EgressOperationStore.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    public interface IEgressOperationStore
```

Comment:
I this interface really necessary? Isn't this an internal implementation detail of the server assembly?

Reply:
It's not that I'm creating a public api here, this is just standard dependency injection into the controller. I can use the class directly, but it will need to remain public.